Writerside/topics/compound-parser.md

```
@@ -0,0 +1,140 @@
+```
+
+</tab>
+
+</tabs>
+
+That is everything you need to do to create your own compound parser.
+There are a couple of things to note:
+ - You need to provide all of your parsers that you will want to use
+ - You may want to create a wrapper class that will wrap your compound parser and will cache the variants so you don't have to create them every time someone want to change keyword/priority
+
+#### How could we warp it?
```

Markdownlint (0.35.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)